Pinta.Effects/Effects/TwistEffect.cs

```diff
@@ -119,17 +128,18 @@ private RenderSettings CreateSettings (ImageSurface dst)
 		);
 	}
 
-	private static PointD[] InitializeAntialiasPoints (int aaLevel)
+	private static ImmutableArray<PointD> InitializeAntialiasPoints (int aaLevel)
```

Comment:
In the future please try to keep these types of changes separate, but it's fine for this PR since I've reviewed it already anyways